src/Tizen.NUI.Components/Controls/Menu.cs

```diff
@@ -55,31 +57,67 @@ protected override void Dispose(DisposeTypes type)
 
             if (type == DisposeTypes.Explicit)
             {
-                if (menuItems != null)
+                if (Content != null)
                 {
-                    foreach (MenuItem menuItem in menuItems)
+                    if (menuItems != null)
                     {
-                        Utility.Dispose(menuItem);
+                        foreach (MenuItem menuItem in menuItems)
+                        {
+                            Content.Remove(menuItem);
+                        }
                     }
+
+                    Utility.Dispose(Content);
                 }
 
-                menuItemGroup = null;
+                Utility.Dispose(Scrim);
 
-                if (anchorPosition != null)
-                {
-                    anchorPosition.Dispose();
-                    anchorPosition = null;
-                }
+                menuItemGroup = null;
 
-                Utility.Dispose(scrollableBase);
+                layer.Remove(this);
+                Window.RemoveLayer(layer);
+                layer.Dispose();
             }
 
             base.Dispose(type);
         }
 
+        /// <summary>The Menu's relative position to Anchor horizontally.</summary>
+        /// <since_tizen> 9 </since_tizen>
+        public enum HorizontalRelativePosition
```

Comment:
I think you can share enum type if you define like below
and you do better to use start/end instead `left,right` if you want to support RTL language
``` c#
public enum RelativePosition
        {
            /// <summary>At the start of the Anchor.</summary>
            /// <since_tizen> 9 </since_tizen>
            Start = 0,
            /// <summary>At the center of the Anchor.</summary>
            /// <since_tizen> 9 </since_tizen>
            Center = 1,
            /// <summary>At the end of the Anchor.</summary>
            /// <since_tizen> 9 </since_tizen>
            End = 2,
        }
```

Reply:
Thank you for your comment.
I have changed as you commented.